runtests.py

```diff
@@ -16,11 +16,24 @@
     "numpy": {
         # Stdlib modules in the namespace by accident
         "absolute_import",
+        "division",
+        "print_function",
         "warnings",
-        # Accidentally public
+        # Accidentally public, deprecated, or shouldn't be used
+        "Tester",
         "add_docstring",
         "add_newdoc",
         "add_newdoc_ufunc",
+        "core",
+        "fastCopyAndTranspose",
+        "get_array_wrap",
+        "int_asbuffer",
+        "oldnumeric",
+        "recfromcsv",
+        "recfromtxt",
```

Comment:
These two `recfrom` functions are the only ones I'm not sure about, they're not deprecated and are regular data loading functions. Why should they be ignored?

Reply:
I think I ignored them because they aren't in the refguide and because of https://github.com/numpy/numpy/issues/12385#issuecomment-509505680. Happy to add them back though.

Reply:
Added the two `recfrom` functions back.